src/main/java/com/ita/if103java/ims/service/impl/InvitationServiceImpl.java

```diff
@@ -63,6 +61,7 @@ private void sendInvitationMessage(UserDto userDto, Long accountId) {
         mailService.sendMessage(userDto, "Hello, We invite you to join our organization " + account.getName() + " in the Inventory Management System.\n" +
             "Please follow link bellow to proceed with registration:\n" +
             activationURL + userDto.getEmailUUID() + "\n" +
+            "Your password: " + userDto.getPassword() + "\n" +
```

Comment:
Please add something like this. _For security purpose please change it as soon as possible._